spec/shared_stripe_examples/customer_examples.rb

```diff
@@ -535,4 +535,38 @@ def gen_card_tk
     customer = Stripe::Customer.retrieve("test_customer_update")
     expect(customer.discount).to be nil
   end
+
+  it "retrieves a stripe customers payment method" do
+    payment_method = Stripe::PaymentMethod.create({
+      type: 'card',
+      card: {
+        number: '4242424242424242',
+        exp_month: 12,
+        exp_year: 2024,
+        cvc: 123
+      }
+    })
+
+    customer = Stripe::Customer.create({
+      email: 'johnny@appleseed.com',
+      description: "a description"
+    })
+
+    payment_method.attach(customer: customer.id)
+
+    retrieved_payment_method = Stripe::Customer.retrieve_payment_method(customer.id, payment_method.id)
+    expect(retrieved_payment_method.id).to eq(payment_method.id)
```

Comment:
Is it feasible/logical to assert that the entire object is the same? That would be a little more compelling of a test.

Reply:
It is feasible to assert that the entire object is the same. I could try it.

Reply:
Seems to work. Going to open a new PR to apply some of the fixes needed.